src/main/java/tay/resource/MessageSendResource.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * 
+ */
+package tay.resource;
+
+import com.twilio.rest.api.v2010.account.Message;
+import com.twilio.type.PhoneNumber;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+import com.twilio.Twilio;
+
+
+/**
+ * @author toby
+ *
+ */
+
+@Path("send")
+public class MessageSendResource {
+	
+	@GET
+	public void sendText() {
```

Comment:
how does a get method end up sending a message considering it doesn't take any input? do i have these reversed?

Reply:
think the indenting is borked here.